File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        boolean answer = true;
        if(s.length() == 4 || s.length() == 6){
            for(int i=0; i<s.length(); i++){
                if(s.charAt(i) >= 65 && s.charAt(i) <=90 ){
                    return false;
                }
                if(s.charAt(i) >= 97 && s.charAt(i) <=122){
                    return false;
                }
            }
        }
        else {
            return false;
        }
        return answer;
    }
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.js
function solution(s) {
    var answer = true;
    if(s.length == 4 || s.length == 6){
        for(let i=0; i<s.length; i++){
            if(s.charAt(i) >= 'a' && s.charAt(i) <= 'z' ){
                return false;
            }
            if(s.charAt(i) >= 'A' && s.charAt(i) <= 'Z'){
                return false;
            }
        }
    }
    else {
        return false;
    }
    return answer;
}